test/scripts/core_enable_disable.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/sh
+#
+# Copyright © 2017-2018 The Crust Firmware Authors.
+# SPDX-License-Identifier: (BSD-3-Clause OR GPL-2.0)
+#
+
+cpu_disable() {
+  echo 0 > "$1"/online
```

Comment:
Do you notice all of the `line 8: echo: write error: Invalid argument` when running this? `echo` adds a newline. The file only wants the number. Replace `echo` with `printf` and I'll approve

Reply:
Strange, I didn't receive that. I'll fix now.

Reply:
Fixed.